src/screens/Authentication/AdminRegister/AdminRegister.tsx

```diff
@@ -1,13 +1,13 @@
 import { React, useState, useContext } from 'react';
-import { TextInput, Text, View, Image, Pressable } from 'react-native';
+import { Text, View, Image, Pressable } from 'react-native';
 // import { useAuthentication } from '../../utils/hooks/useAuthentication';
 import { getAuth } from 'firebase/auth';
-import { registerWithEmailAndPassword } from '../../../firebase/auth';
 import styles from './styles';
 // import FormInput from '../../../components/FormInput/FormInput';
-import FormInput from '../../../components/JobPostFormInput/JobPostFormInput';
+import FormInput from '../../../components/EmailPasswordInput/EmailPasswordInput';
```

Comment:
I believe the component you're importing here is `EmailPasswordInput` and not `FormInput`. Importing it as `FormInput` may cause confusion given that there is another component called `FormInput`.

Reply:
In fact, is the `FormInput` used at all anywhere? If not, we should delete this component.